edk2toolext/environment/var_dict.py

```diff
@@ -138,22 +164,29 @@ def GetValue(self, k, default=None):
     def SetValue(self, k, v, comment, overridable=False):
         """Sets an environment variable to be used throughout the build.
 
+        Additionally able to delete the environment variable if the value is None.
+
         Args:
             k (str): The key to store the value under
-            v (varied): The value to store
+            v (obj | None): The value to store or None to delete
             comment (str): A comment to show where / how the variable was stored.
                 Useful for debugging
             overridable (bool): Specifies if the variable is allowed to be override
                 elsewhere in the build
 
         Returns:
-            (bool): If the variable was successfully stored or not
+            (bool): If the variable was successfully [stored|deleted] or not
         """
         key = k.upper()
         en = self.GetEntry(key)
         value = str(v)
-        self.Logger.debug("Trying to set key %s to value %s" % (k, v))
-        if (en is None):
+
+        if not v:
```

Comment:
This is bad syntax.  What happens if V == 0.

Reply:
see https://peps.python.org/pep-0008/#programming-recommendations for how comparing for none should be handled.